src/components/SmartCollapse/smartCollapses.ts

```diff
@@ -0,0 +1,17 @@
+export type SmartCollapseKey = 'garminInstructions';
+
+type SmartCollapse = {
+  key: SmartCollapseKey;
+  isDefaultCollapsed: boolean;
+};
+
+/**
+ * smartCollapses is a list of collapsing sections in the app
+ * which "remember" their state from local storage
+ */
+export const smartCollapses: SmartCollapse[] = [
+  {
+    key: 'garminInstructions',
+    isDefaultCollapsed: true,
```

Comment:
This is tricky -- because we wait for the local storage value, we have to choose between:

- start open, close once we check local storage
- start closed, open once we check storage (this is what's here now)

Otherwise maybe we look into storing this into a cookie so the server can read it